pythonosc/osc_message_builder.py

```diff
@@ -21,13 +21,14 @@ class OscMessageBuilder(object):
     ARG_TYPE_MIDI = "m"
     ARG_TYPE_TRUE = "T"
     ARG_TYPE_FALSE = "F"
+    ARG_TYPE_NIL = "N"
```

Comment:
you already sent that change in a different PR, please revert

Reply:
Damn, sorry for the mess, I must have had a particularly bad day.

I'll clean this up and try again.